htmldocx/h2d.py

```diff
@@ -315,7 +340,31 @@ def handle_starttag(self, tag, attrs):
         elif tag == 'li':
             self.handle_li()
 
-        elif tag[0] == 'h' and len(tag) == 2:
+        elif tag == "hr":
+
+            # This implementation was taken from:
+            # https://github.com/python-openxml/python-docx/issues/105#issuecomment-62806373
+
+            self.paragraph = self.doc.add_paragraph()
+            pPr = self.paragraph._p.get_or_add_pPr()
+            pBdr = OxmlElement('w:pBdr')
+            insert_element_before(pPr, pBdr, successors=(
```

Comment:
`insert_element_before()` is now included in python-docx ([this comment](https://github.com/python-openxml/python-docx/issues/105#issuecomment-442786431) in the referenced thread).

Reply:
I updated the code to use `pPr.insert_element_before` as mentioned in the comment.